x/bank/keeper/msg_server.go

```diff
@@ -29,19 +30,31 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 func (k msgServer) Send(goCtx context.Context, msg *types.MsgSend) (*types.MsgSendResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	if err := k.IsSendEnabledCoins(ctx, msg.Amount...); err != nil {
-		return nil, err
-	}
-
+	// basic validations start
 	from, err := sdk.AccAddressFromBech32(msg.FromAddress)
 	if err != nil {
```

Comment:
We should improve the error message here by taking the ones from the previous ValidateBasic